docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md

```diff
@@ -41,13 +41,17 @@ A microservice-based application will often use a combination of these communica
 
 These axes are good to know so you have clarity on the possible communication mechanisms, but they are not the important concerns when building microservices. The asynchronous nature of client thread execution not even the asynchronous nature of the selected protocol are the important points when integrating microservices. What *is* important is being able to integrate your microservices asynchronously while maintaining the independence of microservices, as explained in the following section.
 
-## Asynchronous microservice integration enforce microservice’s autonomy
+## Asynchronous microservice integration enforces microservice’s autonomy
 
 As mentioned, the important point when building a microservices-based application is the way you integrate your microservices. Ideally, you should try to minimize the communication between the internal microservices. The less communications between microservices, the better. But of course, in many cases you will have to somehow integrate the microservices. When you need to do that, the critical rule here is that the communication between the microservices should be asynchronous. That does not mean that you have to use a specific protocol (for example, asynchronous messaging versus synchronous HTTP). It just means that the communication between microservices should be done only by propagating data asynchronously, but try not to depend on other internal microservices as part of the initial service’s HTTP request/response operation.
 
 If possible, never depend on synchronous communication (request/response) between multiple microservices, not even for queries. The goal of each microservice is to be autonomous and available to the client consumer, even if the other services that are part of the end-to-end application are down or unhealthy. If you think you need to make a call from one microservice to other microservices (like performing an HTTP request for a data query) in order to be able to provide a response to a client application, you have an architecture that will not be resilient when some microservices fail.
 
-Moreover, having dependencies between microservices (like performing HTTP requests between them for querying data) not only makes your microservices not autonomous. In addition, their performance will be impacted. The more you add synchronous dependencies (like query requests) between microservices, the worse the overall response time will get for the client apps.
+Moreover, having HTTP dependencies between microservices (like creating long request/response cycles with HTTP request chains as shown in the first part of the Figure 4-15), not only makes your microservices not autonomous. In addition, their performance will be impacted. The more you add synchronous dependencies (like query requests) between microservices, the worse the overall response time will get for the client apps.
```

Comment:
not only makes your microservices not autonomous _> this sentence seems to be cut
I'd expect a construction such as `not only ... but also`

suggestion: not only makes your microservices not autonomous but also their performance is impacted.

Avoid having sentences in parentheses and future tense.

suggestion:
The more you add synchronous dependencies between microservices, such as query requests, the worse the overall response time gets for the client apps.

Reply:
OK, updated according your suggestions.